### PR TITLE
#629 Merge upgrader installers to one single universal installers

### LINKED ISSUE #629: Merge upgrader installers into one single full installers
As per discussion regarding building one single universal windows installers for each flavor (desktop and server)
- https://github.com/openmsupply/open-msupply/pull/622#issuecomment-1250612971
- https://github.com/openmsupply/open-msupply/pull/622#issuecomment-1251541273

### PR DESCRIPTION
**Pull request by raviSussol**
Closes #629 

Builds one single universal windows installer of each flavor (Desktop and Server).
- Updated suf files to adjust for both full as well as upgrader installer details
- Removed upprader installers suf files as well removed build bat commands
- Handle registry for storing current omsupply app details

### CODE REVIEW

**Review by mark-prins — COMMENTED** [review]
😁  Nice! that is very neat - I like what you've done there.

One extra suggestion - can you make the window title change based on upgrade / install as well? maybe the text in the welcome screen? Would be great if that could tell the user if it was upgrading or installing. The first welcome screen could be an upgrade message?

**Comment by raviSussol** [comment]
> I think it needs to store the install folder somewhere like the registry, which could be checked when running the installer for existing versions - and skip the folder selection when upgrading. 

Have updated suf files to store settings details in windows registry 14575e3a6

> Also needs to add in the code to stop the service on the 'before install' step if upgrading the server.

This has been handled already.
 
> One extra suggestion - can you make the window title change based on upgrade / install as well? maybe the text in the welcome screen? Would be great if that could tell the user if it was upgrading or installing. The first welcome screen could be an upgrade message?

14575e3a6

**Comment by raviSussol** [comment]
[Merge upgrader installers to one single universal installers](https://trello.com/c/CzjL5ta9/723-merge-upgrader-installers-to-one-single-universal-installers)

**Review by mark-prins — APPROVED** [review]
Didn't get time to test, the changes look great though! Excited to see it in action now! 😃

File: build/windows/omsupply_desktop.suf [hunk 1]
```diff
@@ -3045,7 +3077,28 @@ g_HandleSystemReboot();
...
 <Args/>
-<Script/>
+<Script>local sWindowTitle = "omSupply Desktop Setup";
+local sReadyToInstallSubTitle = SessionVar.Expand("You are now ready to install %ProductName% %ProductVer%.");
+
+hasOmsupplyDesktopInstalled = false;
+omSupplyDesktopPath = SessionVar.Expand("%AppFolder%\\open mSupply.exe");
+
+-- Check if the omSupply Desktop has been already installed on this machine
+if Registry.DoesKeyExist(HKEY_LOCAL_MACHINE, "Software\\omSupply Desktop") then
```
Comment (mark-prins) [inline build/windows/omsupply_desktop.suf:3087]: I tried installing the test-21 installers and the registry keys were not created. are they working ok for you?

**Comment by mark-prins** [comment]
note: ran a new build and works like a charm. This is too cool not to have in, so have merged it! hope that's ok 😁

**Pull request merged**